release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -469,6 +469,14 @@ The following metric has been removed:
 
 * `ovnkube_master_skipped_nbctl_daemon_total`
 
+[id="ocp-4-12-multi-zone-ipi-vsphere-installation"]
+==== Multi-zone Installer Provisioned Infrastructure VMware vSphere installation (Technology Preview)
+Beginning with {product-title} {product-version}, you can configure multiple datacenter, single vCenter installations using Installer Provisioned Infrastructure installation. Using vCenter tags, you can associate vCenter datacenters and compute clusters with OpenShift regions and zones . These associations can help separate failure domains.
```

Comment:
Should OpenShift be product title. Disregard if not.
```suggestion
Beginning with {product-title} {product-version}, you can configure multiple datacenter, single vCenter installations using Installer Provisioned Infrastructure installation. Using vCenter tags, you can associate vCenter datacenters and compute clusters with {product-title} regions and zones . These associations can help separate failure domains.
```

Reply:
Done! Thank you!